sklearn/utils/tests/test_extmath.py

```diff
@@ -249,6 +253,37 @@ def test_randomized_svd_transpose_consistency():
     assert_almost_equal(s2, s3)
 
 
+def test_randomized_svd_power_iteration_normalizer():
+    # randomized_svd with power_iteration_normalized='none' diverges for
+    # large number of power iterations on this dataset
+    X = fetch_olivetti_faces().data[:, :2000]
```

Comment:
We should not force the user to download datasets to run the tests. 2 options:

- find a way to generate random data from a rng to simulate the same behavior.
- make it possible to skip this particular test if the data has not been previously downloaded:

```python
from sklearn.datasets.base import get_data_home, _pkl_filepath

if not exists(_pkl_filepath(get_data_home(), 'olivetti.pkl')):
     raise SkipTest("Skipping dataset loading randomized SVD test")
```

I would prefer the first solution.